frontend/src/component/admin/users/UsersHeader/LicensedUsersBox.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Link, styled, Typography } from '@mui/material';
+import { HelpIcon } from 'component/common/HelpIcon/HelpIcon';
+
+const StyledContainer = styled(Box)(({ theme }) => ({
```

Comment:
Just a suggestion: this can totally be an [`article`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) if you want. Doesn't matter much, though, but it's a standalone component that you could anywhere.